app/employeehandler.py

```diff
@@ -96,21 +96,21 @@ async def get(self, request):
     job_role = employee_info.get('jobRole', "-")
 
     job_role['contractStartDate'] = format_to_uk_dates(
-        job_role['contractStartDate'])
+        job_role.get('contractStartDate'))
```

Comment:
These lines (98-101) seem to be duplicated below, and can be removed.

Reply:
Deleted